src/SDKs/CustomProviders/Microsoft.CustomProviders/Microsoft.CustomProviders.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<Description>Provides developers with libraries to create and manage custom provider resource. Note: This client library is for CustomProviders under Azure Resource Manager.</Description>
```

Comment:
Please move description to `PackageReleaseNotes`

Reply:
Moved to PackageReleaseNotes